src/pages/nina.jsx

```diff
@@ -1,31 +1,36 @@
 import React from "react";
+import bahamas from "/Users/ninabaffo/runtime-terror-website/src/assets/bahamas.jpeg";
+import family from "/Users/ninabaffo/runtime-terror-website/src/assets/family.jpeg";
+import puppy from "/Users/ninabaffo/runtime-terror-website/src/assets/puppy.jpeg";
+import myPic from "/Users/ninabaffo/runtime-terror-website/src/assets/profilePic.png";
```

Comment:
Super close! What you want to do is reverence the picture from a relative directory. Right now you are referencing them from an absolute directory. The server doesn't actually have the folders /Users/ninabaffo/runtime-terror-website so what you want to do is reference them relatively. So in this case it would be `../assets/bahamas.jpeg`

Reply:
Here is a little info on it
https://www.codingrooms.com/blog/file-paths#:~:text=A%20relative%20path%20describes%20the,regularly%20use%20relative%20file%20paths.